### DISCUSSION TITLE
Huge rewrite (+ apology)

### DISCUSSION THREAD

**Discussion by riverpaper** [Pokemon-PythonRed/Pokemon-PythonRed#148] [Announcement]
To all the wonderful members of the PPR community:

Look, I know this project so far has been a hot mess under my leadership. There have been various brilliant contributions by community members, but I want to start seriously reworking some components (translation: I don’t like **my** code).

This was the first GitHub project I ever started, and was meant as a way for me to simultaneously learn Git, GitHub, and Python in the context of something I’m passionate about, being Pokémon (side note, I was also 13 at the time). Even if I don’t have an end goal, I want to do my best to manage this project properly, if only to give our community what they deserve.

Currently, I work on projects such as these from my school computer, which has issues connecting to GitHub (I literally can’t get on the GitHub website, use Git, use VSCode extensions, etc.). Because of this, when I say I’ll “take a look when I get the chance,” it doesn’t happen, and I want to apologise for that.

I have opened #147 as a place for me to rewrite anything I think I need to. Thank you all for your contributions so far, and I’ll try to do my part better.

**Comment by jelleholtkamp** [comment]
Is the rewrite still in progress? I stumbled across this project for another project I had in mind which requires largely the same codebase so I used this project as a form of proof-of-concept. The concept works but I did end up deciding to start from scratch on my own to create a little more efficient baseline code so that expanding the actual game content development can move a little quicker in the future. It's a personal hobby project right now and I don't know if I will end up having the free time I need, but if I do it might be an idea to publish the code here and maybe breathe a little life in this project instead of starting my own repo without a pre-existing community.

**Reply by riverpaper** [reply]
Yeah, I'd definitely like to continue this, but I'm in my last year of high school at the moment, so I'm too busy 😐

Contributions are definitely appreciated, but I'm actually in the process of converting this "proof-of-concept" into a whole text-adventure framework if that helps. After that, I'd keep working on this game and others.

I hope that answers your questions!
